lib/driver.js

```diff
@@ -409,6 +414,14 @@ class AndroidDriver extends BaseDriver {
       log.info('Restoring window animation state');
       await this.adb.setAnimationState(true);
     }
+
+    // This was necessary to change animation scale over Android P
+    // Calls below everytime to keep the api policy safe for the security
+    if (parseInt(this.caps.platformVersion, 10) >= 28) {
```

Comment:
do we want to call it inside `if (this._wasWindowAnimationDisabled) {` block?

Reply:
I'm just concerned about performance. Each new adb command execution will slow down session deletion

Reply:
I see. I can agree with you 👍